File: src/DataClasses/Song.js
class Song {
  constructor(artist, title, url = null, features = null) {
    this.artist = artist;
    this.title = title;
    this.url = url;
    this.features = features;
  }

  isSameSong(song) {
    if (song.title === this.title)
      return true;
    else
      return false;
  }
}

export default Song;

File: MusicTimeApi/MusicTimeApi/Models/Song.cs
namespace MusicTimeApi.Models
{
    public class Song
    {
        public Song(int id, string title, string url, string albumTitle)
        {
            Id = id;
            Title = title;
            URL = url;
            AlbumTitle = albumTitle;
        }

        public int Id { get; }
        public string Title { get; }
        public string URL { get; }
        public string AlbumTitle { get; }
    }
}
